src/main/java/redis/clients/jedis/search/Query.java

```diff
@@ -192,6 +198,9 @@ public void addParams(CommandArguments args) {
     if (_withPayloads) {
       args.add(SearchKeyword.WITHPAYLOADS.getRaw());
     }
+    if (_withSortKeys) {
+      args.add(SearchKeyword.WITHSORTKEYS.getRaw());
```

Comment:
The doc says:

> This is usually not needed by users, and exists for distributed search coordination purposes.

I would prefer to ignore this for now.

Reply:
Ok. Removed the changes related to sortKeys.
Will create a separate PR for this to be included in 5.0.0 release. Anyways we can just give the option for user to use it or not.